src/storyblok-auth-api/grant/grant-handler.ts

```diff
@@ -32,7 +32,7 @@ export const grantHandler =
         storyblok: {
           client_id: clientId,
           client_secret: clientSecret,
-          scope: ['read_content'],
+          scope: scope ? scope : ['read_content'],
```

Comment:
```suggestion
          scope,
```

The scope is a required parameter, so no need to use a fallback value